pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -161,12 +168,13 @@ class CollectionViewSet(
     ViewSet for Collections.
     """
 
-    authentication_classes = []
-    permission_classes = []
     serializer_class = CollectionSerializer
     filterset_class = CollectionFilter
     pagination_class = LimitOffsetPagination
 
+    def urlpattern(*args, **kwargs):
+        return "ansible-collection-viewset"
```

Comment:
I can believe this is some kind of hack, but it looks weird to see the term "viewset" in a url pattern.
(And yes, we use the field `viewset_name` for something resembling a path in the access policy for historical reasons.)

Reply:
I wasn't sure what to name these, so I just came up with something quick and dirty. Please let me know how to make these more pulpy

Reply:
I haven't looked myself, but maybe look at `show-urls` to get a feel for the patterns in place before and after this PR.